### DISCUSSION TITLE
pySigma and sigmac

### DISCUSSION THREAD

**Discussion by FrancescoFaenzi** [SigmaHQ/pySigma#21] [Q&A] [answered]
A bit confused about the two: is pySigma a full replacemente of sigmac?
Should we stop using sigmac and move to pysgma from now on or is it still WIP?
Thank you

**Comment by thomaspatzke** [comment, marked as answer]
pySigma is currently wip and only a library than a converter. Furthermore, only a Splunk backend exists currently in pySigma. The converter part will be developed as separate project based on pySigma. Currently there is a small cli for testing purposes contained in pySigma.

The current sigmatools and sigmac will not be continued and replaced by the new tools. New features and modifiers will not be backported to the old tools

**Reply by FrancescoFaenzi** [reply]
Thank you.
